os/drivers/testcase/testcase_drv.c

```diff
@@ -168,6 +169,12 @@ static int testcase_drv_ioctl(FAR struct file *filep, int cmd, unsigned long arg
 		break;
 #endif
 
+#ifdef CONFIG_AMEBAD_TRUSTZONE
+	case TESTIOC_RDP:
+		ret = test_rdp();
+		break;
```

Comment:
It is driver, it should return one of error in errno.h if ret is not OK.

Reply:
Done